src/components/views/settings/tabs/room/SecurityRoomSettingsTab.tsx

```diff
@@ -29,19 +29,19 @@ import {UIFeature} from "../../../../../settings/UIFeature";
 import { replaceableComponent } from "../../../../../utils/replaceableComponent";
 
 // Knock and private are reserved keywords which are not yet implemented.
-enum JoinRule {
+export enum JoinRule {
```

Comment:
we should move these enums to the js-sdk one day

Reply:
I actually did - https://github.com/matrix-org/matrix-js-sdk/pull/1745/files